Gopkg.toml

```diff
@@ -3,8 +3,8 @@
   name = "code.cloudfoundry.org/lager"
 
 [[constraint]]
-branch = "master"
   name = "github.com/hashicorp/vault"
+  version = "1.0.0"
```

Comment:
It was previously pinned to a specific Vault commit near version 0.4.7 if memory serves. Vault didn’t introduce namespaces until a much later versions, thus it is required to support this feature. Furthermore, you should always pin your dependencies to a specific version.

Reply:
Ah yes, that's true, it was bumped up to pull in the `SetNamespaces` method on the Vault client since it didn't exist in previous versions. I'd be fine with pointing it at master, just did 1.0.0 because it seemed nicer to read.